packages/rancher-kiali-server/package.yaml

```diff
@@ -1,5 +1,5 @@
 url: https://kiali.org/helm-charts/kiali-server-1.24.0.tgz
-packageVersion: 01
+packageVersion: 02
```

Comment:
@brendarearden I don't think it makes a difference w.r.t. the packaging of the chart since the `requirements.yaml` uses a local chart archive, but should we be updating https://github.com/rancher/charts/blob/70bcfcbee35caff374021bcc1fd0cb721f766640/packages/rancher-istio/charts/requirements.yaml#L6 with the correct `rancher-kiali-server` version (including the packageVersion)?

Reply:
Note for future: This is a weird corner case of our charts scripts that needs to be addressed in https://github.com/rancher/rancher/issues/28730.

Reply:
I thought that the base version stayed the same, and we were appending the extra `000` or `001` on the end - is that not what the package version is for? Since we are just adjusting what we patch over the top of the 1.24.0 version, I think that requirements version should stay the same, but https://github.com/rancher/charts/blob/948c7b8f9a7a847eca3d680271ee0c2da11ddde5/packages/rancher-istio/charts/Chart.yaml#L16 would update... at least this was my understanding

Reply:
Yeah I think that makes sense. That's how we've done it so far so probably that shouldn't change.

For this PR, I don't think we need to necessarily make that change (or rather shouldn't, to keep it consistent with the prior release) but it's definitely weird that our Helm packages point to a non-existent chart in the `requirements.yaml` when it should also be pointing to the actual version that the annotation points to. 

e.g. the file we point to in the `requirements.yaml` doesn't even exist in the `main` branch 

https://github.com/rancher/charts/blob/828fdd507ae0d5be0b6f017f39aff6742356c362/charts/rancher-istio/requirements.yaml#L6-L7

Something to fix in the charts script rewrite, but not something that needs to be addressed here 😅

Reply:
yeah that is very weird - I see what you are saying now.